elements/urth-core-channels/urth-core-channels.html

```diff
@@ -105,9 +105,11 @@
             console.debug('urth-core-channels onModelChange', options);
 
             var changes = options.changed;
-            Object.keys(changes).forEach(function(change){
+            Object.keys(changes).filter(function(change){
+                return !(change === "status");
```

Comment:
Agreed on changing it to `__status__`, I think that `update` might be needed to trigger the Backbone change (which triggers `onModelChange`) but I'm not 100%